runtime/app/xwalk_main_delegate.h

```diff
@@ -35,12 +36,13 @@ class XWalkMainDelegate : public content::ContentMainDelegate {
   content::ContentBrowserClient* CreateContentBrowserClient() override;
   content::ContentRendererClient* CreateContentRendererClient() override;
 
-  static void InitializeResourceBundle();
```

Comment:
The member `"resource_delegate_"` is added and referenced in member-function `InitializeResourceBundle()`.  So it cannot be static unless we define` "resource_delegate_"` as static too.  So I refered the similar usage from chromecast/app/cast_main_delegate.h